### DISCUSSION TITLE
Character variant of ẞ using the Frankfurt shape?

### DISCUSSION THREAD

**Discussion by soc** [rsms/inter#517] [Q&A]
Thank you for all the work you have done @rsms, I absolutely love your font!

The current ẞ in Inter uses the Dresden shape (no. 1 in the picture), which is generally the most popular variant and works well for most font designs.

Given Inter's rather geometric shape though, I believe that a character variant of ẞ using the Frankfurt shape (no. 3 in the picture) would be a great addition that followed Inter's overall feel.

![versaleszett_formen](https://user-images.githubusercontent.com/42493/200189179-95778c1a-ea26-4c35-b5ce-b8d0c3f0d91f.png)

I'm wondering whether it's possible to add such a character variant of ẞ to Inter – either via contribution or money?

Thanks,

Simon

---

Further references:
- https://typography.guru/journal/capital-sharp-s-designs/
- https://finaltype.de/de/themen/versal-scharf-s
- https://typefacts.com/blog/das-grosse-eszett (source of the picture above)

**Comment by rsms** [comment]
@soc What about something fun in between like this?

<img width="404" alt="Screen Shot 2024-03-26 at 14 14 06" src="https://github.com/rsms/inter/assets/47046/835bffb1-74d3-4131-81c2-2487feff67c8">

<img width="822" alt="Screen Shot 2024-03-26 at 14 12 57" src="https://github.com/rsms/inter/assets/47046/ce07f9f0-a649-40b6-bf69-6f315e424da1">

**Reply by soc** [reply]
A while ago I found this info sheet that has some interesting suggestions (some which I disagree with), perhaps it's still useful:

![EIyNBBfW4AY09nm](https://github.com/rsms/inter/assets/42493/56e8a67b-4fb3-49cc-9d38-ed92981fe2e5)

**Reply by soc** [reply]
(Please let me know if you aren't interested in tweaking this any further, then I'll stop.)

Having thought about your design, two things that came to my mind.

1. The letter feels a bit "top-heavy"; I think this has multiple causes:

- The "middle bar" feels a bit too high; looking at `STRAẞE` above, it feels like it should be at height of the middle bar of the E at max, perhaps even lower.
- Receding the upper right corner a bit might make the angle be less "aggressive".

2. Distinctiveness from "B":

- The tail could point slightly upward, to make it look less like a "B". (I understand this could not be worth the trouble considering the trouble it might cause in bigger weights.)

**Comment by soc** [comment]
Though I'm kinda partial to shape number 3, your variant also fits and looks pretty!
I think it might even be a great design to be the default in your font!

(I love Sofia Sans' take on variant 3, that's kinda where I'm coming from:)
![straße](https://github.com/rsms/inter/assets/42493/c5dfceb8-530a-4601-8ce8-ca079101162a)

**Comment by rsms** [comment]
Made the new design the default (the old one available via cv14)
https://github.com/rsms/inter/issues/693#issuecomment-2021551424

![](https://github.com/rsms/inter/assets/47046/bdfe89d9-55ea-4977-b5f8-5c4f0597d643)

**Reply by soc** [reply]
Niceeeee! :-)
